chain/actors/builtin/builtin.go

```diff
@@ -65,9 +67,8 @@ func QAPowerForWeight(size abi.SectorSize, duration abi.ChainEpoch, dealWeight,
 }
 
 func ActorNameByCode(c cid.Cid) string {
```

Comment:
We really should be merging this code with the bundle code. Exporting two sets of methods for looking up actor names where one only works after v8 is strange. But I'm planning on leaving that to a different PR.